src/datasources/human-description-api/json/contract-descriptions.json

```diff
@@ -0,0 +1,13 @@
+{
+  "function transfer(address, uint256)": "Send {{tokenValue 1}} to {{address 0}}",
+  "function approve(address, uint256)": "Approve {{tokenValue 1}} to {{address 0}}",
+  "function withdraw(uint256)": "Withdraw {{tokenValue 0}}",
+  "function changeThreshold(uint256)": "Change threshold to {{decimals 0}}",
+  "function addOwnerWithThreshold(address,uint256)": "Add new owner {{address 0}} with threshold {{decimals 1}}",
+  "function removeOwner(address,address,uint256)": "Remove owner {{address 1}} with threshold {{decimals 2}}",
+  "function swapOwner(address,address,address)": "Swap owner {{address 1}} with {{address 2}}",
+  "function setGuard(address)": "Set guard {{address 0}}",
+  "function enableModule(address)": "Enable module {{address 0}}",
+  "function setFallbackHandler(address)": "Set fallback handler {{address 0}}",
+  "function approveHash(bytes32)": "Approve hash {{identifier 0}}"
```

Comment:
As I already mentioned before, `0` and `1` don't scream "positional arguments" to me. They look like arguments to the preceding symbol. E.g. `decimals 0` reads like `format number with zero decimal points`.
`$0` and `$1` would be a more intuitive notation.

Reply:
Thats a good point. I will adjust it accordingly.